### ISSUE TITLE
KDE screenlocker in dom0 does not let me type any password

### ISSUE THREAD

**Issue by Rudd-O** [QubesOS/qubes-issues#2664]
Fresh R3.2.

After installing KDE as per #2663 : 

The screen lock password field is greyed out and cannot be clicked, or receive input in any way.  This makes it impossible to break into the system (no VTs getty anymore since R3.2 that I noticed).

Some hints around the Web mention something about PAM configuration.

This is the kscreensaver PAM.d file:

```
#%PAM-1.0
auth       include	system-auth
account    include	system-auth
password   include	system-auth
session    include	system-auth
```

This is the kde PAM.d file:

```
#%PAM-1.0
#auth [user_unknown=ignore success=ok ignore=ignore default=bad] pam_securetty.so
auth       substack     system-auth
auth       include      postlogin
account    required     pam_nologin.so
account    include      system-auth
password   include      system-auth
# pam_selinux.so close should be the first session rule
session    required     pam_selinux.so close
session    required     pam_loginuid.so
session    optional     pam_console.so
# pam_selinux.so open should only be followed by sessions to be executed in the user context
session    required     pam_selinux.so open
session    required     pam_namespace.so
session    optional     pam_keyinit.so force revoke
session    include      system-auth
session    include      postlogin
#-session   optional     pam_ck_connector.so
```

This is the xscreensaver PAM.d configuration:

```
#%PAM-1.0

# Fedora Core 5:
auth       include	system-auth

# SuSE 9.0: (along with "configure --with-passwd-helper" and "unix2_chkpwd")
# auth     required       pam_unix2.so	nullok

# Distant past:
# auth       required	/lib/security/pam_pwdb.so shadow nullok

# Account validation
account include system-auth
```

I will report on which one works.  It is known that the kscreensaver one does not work.

**Comment by Rudd-O** [comment]
The xscreensaver one works.

**Comment by Rudd-O** [comment]
Fix for the screensaver in #2663.  Workaround, really.

**Comment by andrewdavidwong** [comment]
This issue is being closed because:

 - This issue has the "Release 3.2 updates" milestone.
 - [Qubes OS 3.2 recently reached end-of-life (EOL).](https://www.qubes-os.org/news/2019/03/28/qubes-3-2-has-reached-eol/)
 - This issue has not been updated in over a year.

If anyone believes that this issue should be reopened, please let us know in a comment here.
